ui/panels-plugin/src/plugins/line-chart/tooltip/SeriesInfo.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box, Theme } from '@mui/material';
+import { SxProps } from '@mui/system/styleFunctionSx/styleFunctionSx';
+import SeriesMarker from './SeriesMarker';
+
+const seriesNameStyle: SxProps<Theme> = {
```

Comment:
Looks like this is only used in one place. Any reason not to just inline it below where it's used?

Reply:
No good reason, I changed to inline style for this and in the other instance